portfolio/frontend/src/containers/CreateCompetition/CreateCompetition.module.css

```diff
@@ -58,23 +54,49 @@
 	margin-top: 5%;
 }
 
+.CreateCompetitionPage {
+	margin: auto;
+}
+
 .ButtonContainer {
 	width: 50%;
-	height: 30%;
+	height: 100%;
 	display: flex;
 	flex-direction: row;
-	justify-content: space-between;
+	justify-content: space-evenly;
 	margin-top: 10%;
 	margin-left: 27%;
 }
 
-.button {
-	width: 60%;
-	height: 40%;
+.BlueButton {
```

Comment:
Can the common attributes for Blue and White button be factored out

Reply:
I will look into that

Reply:
fix this in 0bf80d684ad4b8dad4d9ba9bbb2ecd87a0e53494

Reply:
very nice